go.sum

```diff
@@ -1,5 +1,6 @@
 github.com/google/uuid v1.1.2 h1:EVhdT+1Kseyi1/pUmXKaFxYsDNy9RQYkMWRH68J/W7Y=
 github.com/google/uuid v1.1.2/go.mod h1:TIyPZe4MgqvfeYDBFedMoGGpEw/LqOeaOT+nhxU+yHo=
 github.com/google/uuid v1.2.0/go.mod h1:TIyPZe4MgqvfeYDBFedMoGGpEw/LqOeaOT+nhxU+yHo=
+github.com/google/uuid v1.3.0/go.mod h1:TIyPZe4MgqvfeYDBFedMoGGpEw/LqOeaOT+nhxU+yHo=
```

Comment:
The release notes indicate that the new version introduces an optional randomness pool mode for V4 UUID generation. While this can improve performance, it may also introduce security concerns, especially in security-sensitive applications. Make sure to review the usage of UUIDs in your application and consider whether to enable this feature based on your security requirements.